### PR TITLE
feature: add getter function for supernode cdn file path

### PR DESCRIPTION
**Pull request by Starnop**
Signed-off-by: Starnop <starnop@163.com>

### Ⅰ. Describe what this PR did
Supernode will store the downloaded files as a cache with CDN. And this pull request define some file paths that CDN will use. 

### Ⅱ. Does this pull request fix one issue?

fixes part of #348

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)

Added.

### CODE REVIEW

File: supernode/config/global_variable.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+var (
+	// HTTPSubPath is the name of the parent directory where the upload files are stored.
+	HTTPSubPath = "qtdown"
+
+	// DownloadSubPath is the name of the parent directory where the downloaded files are stored.
+	DownloadSubPath = "download"
+
+	// DownloadHome is the parent directory where the downloaded files are stored
+	// which is a relative path.
+	DownloadHome = path.Join("/repo", DownloadSubPath)
```
Comment (lowzj) [inline supernode/config/global_variable.go]: Don't use `/repo` as the root directory, it should be configured by user.
There's no necessary to add `/` here, it's different in different operating system.

Comment (Starnop) [reply supernode/config/global_variable.go]: Currently the baseDir is supported user-defined. As for `DownloadHome` and `UploadHome`, they will be used as the bucket. HDYT?

Comment (lowzj) [reply supernode/config/global_variable.go]: It's better to remove the prefix `/` if it's a relative path.

**Comment by Starnop** [comment]
Implement it in the pull request #508 , so close it now.

File: supernode/config/global_variable.go (hunk 1 above)
Comment (Starnop) [reply supernode/config/global_variable.go]: Yeah, that's what I did in the pull request #508.

**Pull request closed without merging**